meinberlin/apps/plans/assets/plans_map.jsx

```diff
@@ -360,24 +360,26 @@ class PlansMap extends React.Component {
   }
 
   renderListItem (item, i) {
-    let className = 'list-item list-item--squashed'
+    let itemClass = 'list-item list-item--squashed'
     if (i === this.state.selected) {
-      className += ' selected'
+      itemClass += ' selected'
+    }
+    let statusClass = 'list-item__status--active'
+    if (item.participation_string === item.participation_display) {
```

Comment:
I find all this logic a bit distributed and well hidden.
Can we use the view to get a status string and the statusClass?